TESTING.md

```diff
@@ -522,3 +522,8 @@ Repeat the above starting from different themes.
 #### Expected behaviour
 
 Changing theme should lead to full repaint of the colours according to the current theme.
+
+### Guardrails:
+
+- [ ] Check if Guardrails check passes for any code longer than 10 lines generated in Cody Chat
```

Comment:
Hey, thank for adding test scenario for Guardrails but it looks a bit incomplete to me.
It does not explain what `Guardrails check` is and how to check if it passed.
Ideally it should be in the form:
1. Do <describe exact steps> to trigger some action
2. Check if results looks like that: <describe expected result>

Reply:
I tried to put in a detailed description of actions, and tangible expectation. Let me know if that works OK.